content/docs/command-reference/checkout.md

```diff
@@ -69,6 +68,9 @@ be pulled from remote storage using `dvc pull`.
 
 ## Options
 
+- `--summary` - displays summary of the changes done by this command in the
```

Comment:
I would make it a bit explicit, I think even in the default mode we output a summary, so it's a shorter version ...

Reply:
No summary is shown without this option. But I did improve it now.